### PR TITLE
Added changes to readme with migration to dragen 3.7.5 and cwl

### PR DESCRIPTION
**Pull request by alexiswl**
# Changelog

* Updated dragen hashtable builder instructions with migration to dragen 3.7.5 and use of CWLtool to build the hashtable.   
* Updated dragen germline workflow logic with changes to dragen 3.7.5 and use of CWLWorkflow

## TODO list

- [ ]  Add output json of successful germline workflow
- [ ]  Add output json of successful somatic workflow 
- [ ]  Add pointer to hashtable builder cwltool (waiting for https://github.com/umccr-illumina/cwl-iap/pull/179)
- [ ] Add pointer to germline workflow (waiting for https://github.com/umccr-illumina/cwl-iap/pull/181)
- [ ] Add pointer to somatic workflow (waiting for https://github.com/umccr-illumina/cwl-iap/pull/182)

### CODE REVIEW

**Review by brainstorm — APPROVED** [review]
Looks good to me so far :)

TL;DR this is a s/GRC37h/hg38/ work to index that genome for Dragen ,right? What's the "todo JSON output" you mention? The resulting output from that indexing action?

**Comment by alexiswl** [comment]
> What's the "todo JSON output"?

I was updating the somatic and germline workflow use cases.   
I have created the inputs but are waiting to merge across some pull requests before completing the outputs and locations on the repo.

**Comment by skanwal** [comment]
@alexiswl LGTM as well. 
Any particular reason to include links to CWL workflows and inputs here? 
Is this document being referenced anywhere?

**Comment by alexiswl** [comment]
@skanwal b07c88b uses prebuilt references instead!

Linked to: https://github.com/umccr/cwl-ica/pull/12
